README.md

```diff
@@ -1 +1,32 @@
-# demo-java
\ No newline at end of file
+<p align="center">
+  <img src="https://deepsource.io/images/logo-wordmark-dark.svg" />
+</p>
+
+<p align="center">
+  <a href="https://deepsource.io/docs/">Documentation</a> |
+  <a href="https://deepsource.io/signup/">Get Started</a> |
+  <a href="https://discuss.deepsource.io/">Discuss</a>
+</p>
+
+<p align="center">
+  DeepSource helps you ship good quality code.
+</p>
+
+</p>
+
+---
+
+# Demo repository - Java
+
+This repository demonstrates sample issues in Java code raised by DeepSource.
+
+[![DeepSource](https://static.deepsource.io/deepsource-badge-light-mini.svg)](https://deepsource.io/gh/deepsourcelabs/demo-java/?ref=repository-badge)
```

Comment:
I believe there is a newer abd apparently better badge available, talk to @rahul-deepsource  regarding that.

Reply:
added the new badge!